dotnet/src/Microsoft.Agents.AI.DevUI/Microsoft.Agents.AI.DevUI.Frontend.targets

```diff
@@ -0,0 +1,68 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- Frontend paths - pointing to the Python package's frontend -->
+    <FrontendRoot>$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)\..\..\..\python\packages\devui\frontend'))</FrontendRoot>
+    <FrontendBuildOutput>$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)\..\..\..\python\packages\devui\agent_framework_devui\ui'))</FrontendBuildOutput>
+    <FrontendPackageJson>$(FrontendRoot)\package.json</FrontendPackageJson>
+    <FrontendNodeModules>$(FrontendRoot)\node_modules</FrontendNodeModules>
+  </PropertyGroup>
+
+  <!-- Ensure npm packages are installed before building -->
+  <Target Name="EnsureNodeModules" BeforeTargets="BeforeBuild" Condition="!Exists('$(FrontendNodeModules)')">
+    <Exec Command="npm install" WorkingDirectory="$(FrontendRoot)" />
```

Comment:
When building from this branch, I had to comment out this and the `npm run build` commands to get the build to work. Perhaps I'm missing a pre-req? (I did build the python bits first). But commenting the lines out, the devui built and seemed to be working.